### ISSUE TITLE
Incorrect casing in spanish

### ISSUE THREAD

**Issue by Carlos-err406** [iamkun/dayjs#2502]
**Describe the bug**
when using the dayjs/locale/es, months are being displayed lowercased, this is wrong, month names must be Uppercase since they are names.
**Expected behavior**
month names shown in uppercase

**Information**
 - Day.js Version 1.11.10
 - OS: WSL
 - Browser: Edge
 - Time zone: irrelevant

**Comment by DeLm0re** [comment]
I also want to point out that it is the same for **dayjs/locale/fr** and it is annoying.

I understand the need for people to have month names in lowercase as much as having them in uppercase. I don't think there is a real choice to make between both but the best solution will be to let the user choice.

**Comment by Carlos-err406** [comment]
> I think by "uppercase" we mean "first letter in uppercase" like in **dayjs/locale/en**
> 
> I also want to point out that incorrect casing is also the case for **dayjs/locale/fr** and it is annoying.
> 
> I understand the need for people to have month names in lowercase as much as having them in uppercase. I don't think there is a real choice to make between both but the best solution will be to let the user choice.

yes, sorry, I meant capitalized

**Comment by Telsho** [comment]
Yep this is the same issue in Italian. All latin languages do not have upper case letters?
